src/commands/Language/sim.js

```diff
@@ -2,7 +2,7 @@ const config = require('../../../config');
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageEmbed } = require('discord.js');
 const db = require('../../index').db;
-const nlp = require('compromise');
+const nlp = require('compromise/one');
```

Comment:
Using comprimise 2 or 3 is too resource intensive. We can only reasonably use 1.

Reply:
You could say we have to _compromise_

Reply:
but lets not say that